apps/admin/backend/src/server.ts

```diff
@@ -761,117 +519,53 @@ export function buildApp({
-    const electionRecord = store.getElection(electionId);
-
-    if (!electionRecord) {
-      return response.status(404).json({
-        status: 'error',
-        errors: [
-          {
-            type: 'not-found',
-            message: `No election found with id ${electionId}`,
-          },
-        ],
-      });
-    }
-
-    const parseBodyResult = safeParse(
-      Admin.PostPrintedBallotRequestSchema,
-      request.body
-    );
-
-    if (parseBodyResult.isErr()) {
-      return response.status(400).json({
-        status: 'error',
-        errors: [
-          {
-            type: 'invalid-value',
-            message: parseBodyResult.err().message,
-          },
-        ],
-      });
-    }
-
-    const id = store.addPrintedBallot(electionId, parseBodyResult.ok());
-
-    response.json({ status: 'ok', id });
-  });
-
-  deprecatedApiRouter.get<
-    { electionId: Id },
-    Admin.GetPrintedBallotsResponse,
-    Admin.GetPrintedBallotsRequest
-  >('/admin/elections/:electionId/printed-ballots', (request, response) => {
-    const { electionId } = request.params;
-    const electionRecord = store.getElection(electionId);
-
-    if (!electionRecord) {
-      return response.status(404).json({
-        status: 'error',
-        errors: [
-          {
-            type: 'not-found',
-            message: `No election found with id ${electionId}`,
-          },
-        ],
-      });
-    }
-
-    const parseQueryResult = safeParse(
-      Admin.GetPrintedBallotsQueryParamsSchema,
-      request.query
-    );
+      ];
+    },
 
-    if (parseQueryResult.isErr()) {
-      return response.status(400).json({
-        status: 'error',
-        errors: [
-          {
-            type: 'invalid-value',
-            message: parseQueryResult.err().message,
-          },
-        ],
+    getPrintedBallots(
+      input: { ballotMode?: Admin.BallotMode } = {}
+    ): Admin.PrintedBallotRecord[] {
+      return store.getPrintedBallots(loadCurrentElectionIdOrThrow(workspace), {
+        ...input,
       });
-    }
-
-    const printedBallots = store.getPrintedBallots(
-      electionId,
-      parseQueryResult.ok()
-    );
+    },
 
-    response.json({ status: 'ok', printedBallots });
+    addPrintedBallots(input: { printedBallot: Admin.PrintedBallot }): string {
+      return store.addPrintedBallot(
+        loadCurrentElectionIdOrThrow(workspace),
+        input.printedBallot
+      );
+    },
   });
+}
 
-  app.use(deprecatedApiRouter);
+/**
+ * A type to be used by the frontend to create a Grout API client
+ */
+export type Api = ReturnType<typeof buildApi>;
 
+/**
+ * Builds an express application.
+ */
+export function buildApp({
+  auth,
+  workspace,
+  logger,
+}: {
+  auth: DippedSmartCardAuthApi;
+  workspace: Workspace;
+  logger: Logger;
+}): Application {
+  const app: Application = express();
+  const api = buildApi({ auth, workspace, logger });
+  app.use('/api', grout.buildRouter(api, express));
   return app;
 }
```

Comment:
Could move `buildApi`, or have a separate `app` file cc @jonahkagan.

Reply:
Motivation for changing would just be to be the same across app backends. I didn't know VxScan was actually doing it the same way, except that it has a separate `server.ts` and `app.ts` file. Will rearrange to be the same.